surfsense_backend/app/agents/researcher/nodes.py

```
@@ -0,0 +1,476 @@
+                user_id=user_id,
+                search_space_id=search_space_id,
+                db_session=db_session,
+                connectors_to_search=connectors_to_search
+            )
+            
+            # Fallback if no documents found
+            if not relevant_documents:
+                print(f"No relevant documents found for section: {section_title}")
+                relevant_documents = [
+                    {
+                        "content": f"No specific information was found for: {question}" 
```

Ruff (0.8.2)

Dictionary comprehension uses static key: `"content"`

(B035)